sigle/src/modules/editor/components/SlateEditor.tsx

```diff
@@ -594,52 +594,66 @@ export const SlateEditor = ({
   };
 
   return (
-    <React.Fragment>
-      <FixedContainer>
-        <AppBar>
-          <StyledAppBarRightContainer>
-            {loadingSave && (
-              <ButtonOutline style={{ marginRight: 6 }} disabled>
-                Saving ...
-              </ButtonOutline>
-            )}
-            {!loadingSave && story.type === 'public' && (
-              <ButtonOutline
-                style={{ marginRight: 6 }}
-                onClick={() => handleSave()}
-              >
+    <Container>
+      <Flex
+        as="header"
+        justify="between"
+        align="center"
+        css={{
+          mt: '$5',
+
+          '@md': {
+            mt: '$10',
+          },
+        }}
+      >
+        <Flex gap="10" align="center">
+          <Link href="/" passHref>
+            <a>
+              <ArrowLeftIcon />
+            </a>
+          </Link>
+          <Text css={{ color: '$gray11' }} size="sm">
+            <Box as="span" css={{ fontWeight: 'bold' }}>
```

Comment:
was having a look at the Figma file and it seems that the name and "Draft" have a different size no?

![Screenshot 2022-01-26 at 15 27 40](https://user-images.githubusercontent.com/5749437/151214661-816093df-1925-44eb-b595-a50661c881cb.png)